### DISCUSSION TITLE
Added support for the Voxelab Aquila S2 printer and plastic profiles for it

### DISCUSSION THREAD

**Discussion by noyhay1** [OrcaSlicer/OrcaSlicer#4901] [Dev]
Added support for the Voxelab Aquila S2 printer and plastic profiles for it
I did this for the first time, test prints were made
@SoftFever 
[resources.zip](https://github.com/SoftFever/OrcaSlicer/files/14894597/resources.zip)

**Comment by SoftFever** [comment]
Hi
Thanks.
Could you submit it via Pull Request?

**Reply by noyhay1** [reply]
I just registered, I don’t know how to register validator
